src/components/bs5/callToAction/callToAction.hbs

```diff
@@ -0,0 +1,9 @@
+<a class="qld-cta-link {{getClassNames "small, view-all" class}}"
```

Comment:
so is this taking the current list or just apending them if the config array is boolean enabled? 
is this more for squiz to be able to drive it with a boolean change but still in the list instead of whats included?

should we also handle a json 'class' object as a possible text that is just a space delimited list?